pytri/__init__.py

```diff
@@ -371,13 +373,19 @@ def graph(self, data, radius: Union[float, Sequence[float]] = 0.15,
             data = json_graph.node_link_data(data)
         _js = self._fetch_layer_file("ColorGraphLayer.js")
 
-        mult_radius = radius * 100
+        PARTICLE_RADIUS_SCALE = 50
+
+        if mesh_nodes:
+            radius = radius
```

Comment:
This is kinda weird — more readable to do `if not mesh_nodes: radius *= PARTICLE_RADIUS_SCALE`